ci/ReleaseTool/PrepCommand.cs

```diff
@@ -32,19 +27,6 @@ internal class PrepCommand
         private const string branchAnnotationTemplate = "* Successfully created a [release candidate branch]({0}) " +
             "in the repo `{1}`, and it will evantually become `{2}` (no pull request as the specified release branch did not exist for this repository).\n";
```

Comment:
```suggestion
            "in the repo `{1}`, and it will eventually become `{2}` (no pull request as the specified release branch did not exist for this repository).\n";
```
Ultranit